src/domain_abstract/ui/InputColor.js

```diff
@@ -44,7 +44,7 @@ export default Input.extend({
     colorEl.get(0).style.backgroundColor = valueClr;
 
     // This prevents from adding multiple thumbs in spectrum
-    if (opts.fromTarget) {
+    if (opts.fromTarget || (opts.fromInput && !opts.avoidStore)) {
```

Comment:
We don't need this anymore :)

Reply:
Actually, it fixes another bug which I described in issue #2683. 
That bug is different from your bug. It happens when user changes color from input, not from color picker.
I also attempt to revert that line, but that bug is still happening.

Reply:
Ah, is it still happening on your side? I've tried with the same bg gradient plugin and seems to be fixed

Reply:
ah ok nvm, you're right, I didn't try the input thing 👍